pkgs/applications/graphics/mcomix3/default.nix

```diff
@@ -56,13 +56,15 @@ python3.pkgs.buildPythonApplication rec {
   postInstall = ''
     rmdir $libdir/mcomix/mcomix
     cp man/* $out/share/man/man1/
+    mv $out/share/man/man1/mcomix.1 $out/share/man/man1/${pname}.1
```

Comment:
Please use installManPage.

Reply:
But that wouldn't fix the name, and I'd need the `mv` with the hardcoded path anyway.

Reply:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/setup-hooks/install-shell-files.sh

Reply:
btw are there multiple man pages? if not cp and mv can be reduced to only mv.

Reply:
There are two. One for the thumbnailer and one for the viewer.